ObjObsSAP.tex

```diff
@@ -256,7 +255,7 @@ \subsection{\{query\} resource} \label{sec:query}
 names refer to those defined in the ObsCore Data Model .
 
 Some of the parameters proposed in this standard are not described in
-the ObsCore Data Model document , such as; min\_vis, max\_vis,
+the ObsCore Data Model document , such as; min\_obs, max\_obs,
```

Comment:
There should probably be a mention about the fact that the name of these columns has changed since the last version of this document. So, it probably worth to mention in the Change Log.